reactNativeProject/src/screens/Main/components/TodoList/components/checkbox/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { View, TouchableOpacity } from "react-native";
+import PropTypes from "prop-types";
+import styles from "./styles";
+
+function Checkbox(props) {
+  const { onToggle, isChecked } = props;
+
+  return (
+    <TouchableOpacity onPress={onToggle}>
+      <View style={styles.box}>
+        {isChecked && <View style={styles.inner} />}
+      </View>
+    </TouchableOpacity>
+  );
+}
+
+Checkbox.propTypes = {
+  onToggle: PropTypes.func,
```

Comment:
This prop is required, since this component would break if it doesn't receive a handler